packages/api/src/queue-processor.ts

```diff
@@ -153,8 +153,12 @@ const main = async () => {
     subscribers: [__dirname + '/events/**/*{.js,.ts}'],
     namingStrategy: new SnakeNamingStrategy(),
     logger: new CustomTypeOrmLogger(['query', 'info']),
-    connectTimeoutMS: 40000, // 40 seconds
+    connectTimeoutMS: 10000, // 10 seconds
     maxQueryExecutionTime: 10000, // 10 seconds
+    extra: {
```

Comment:
are we losing this `options: process.env.PG_EXTRA_OPTIONS,` ? Right now we use that to set the lock timeout.  its set like this in k8s `PG_EXTRA_OPTIONS: '-c lock_timeout=500ms'`

Reply:
> are we losing this `options: process.env.PG_EXTRA_OPTIONS,` ? Right now we use that to set the lock timeout. its set like this in k8s `PG_EXTRA_OPTIONS: '-c lock_timeout=500ms'`

no, we won't but i noticed we didn't have it for queue-processor in the first place. did you do it on purpose?

Reply:
the option was set in api-backend only

Reply:
Oh, yeah I think we will want it everywhere

Reply:
Currently i think the AppEngine queue-processor is using it

Reply:
but it is not in the code

Reply:
ohh, you set it on cloud sql

Reply:
@jacksonh i have removed the `setOptions` in queue-processor